.ci.yaml

```diff
@@ -293,6 +293,10 @@ targets:
       add_recipes_cq: "true"
       release_build: "true"
       config_name: linux_host_engine
+      dependencies: >-
+        [
+          {"dependency": "goldctl", "version": "git_revision:f808dcff91b221ae313e540c09d79696cd08b8de"}
```

Comment:
FYI, this is getting bumped in https://github.com/flutter/engine/pull/47391 to 720a542f6fe4f92922c3b8f0fdcc4d2ac6bb83cd

Reply:
Thanks!

Reply:
Should the Linux one also be the same hash?

Reply:
I looked at the PR and see that it should be.

Reply:
Thanks!